packages/web-forms/vite.config.ts

```diff
@@ -144,34 +144,9 @@ export default defineConfig(({ mode }) => {
 			preprocessorOptions: {
 				scss: {
 					api: 'modern',
+					quietDeps: true, // Suppress warnings from node_modules
 				},
 			},
-			postcss: {
```

Comment:
I forgot to remove this from the PrimeVue upgrade to v4. 

This code was needed for the deprecated library: primevue-sass-theme
And the primevue in this project, doesn't work with CSS layers anymore